src/tests/bitsets/test_stdlib_bitset_large.f90

```diff
@@ -0,0 +1,1504 @@
+                error stop procedure // ' against expectations set0 has ' // &
+                    'at least 1 bit unset.'
+            end if
+        end do
+
+        write(*,*) 'As expected set1 had all bits set.'
+
+        if ( set0 % bits() == 33 ) then
+            write(*,*) 'set0 has 33 bits as expected.'
+        else
+            error stop procedure // 'set0 unexpectedly does not have 33 bits.'
+        end if
+
+! > 64 bit inquiries
+        call set10 % from_string( bitstring_0 // bitstring_0 // bitstring_0 )
+        if ( set10 % none() ) then
+            if ( .not. set10 % any() ) then
+                write(*,*) 'As expected set10 has no bits set'
+            else
+                error stop procedure // ' set10 had some bits set which ' // &
+                    'was unexpected.'
+            end if
+        else
+            error stop procedure // ' set10 did not have none set which ' // &
+                'was unexpected'
+        end if
+
+        call set10 % not()
+
+        if ( set10 % all() ) then
+            if ( set10 % any() ) then
+                write(*,*) 'As expected set10 now has all bits set'
+            else
+                error stop procedure // ' set10 had no bits set which ' // &
+                    'was unexpected.'
+            end if
+        else
+            error stop procedure // ' set10 did not have all bits set ' // &
+                'which was unexpected'
+        end if
+
+        call set11 % from_string( bitstring_all // bitstring_all // &
+            bitstring_all )
+        if ( set11 % any() ) then
+            if ( set11 % all() ) then
+                write(*,*) 'As expected set11 has all bits set'
+            else
+                error stop procedure // ' set11 did not have all bits set ' // &
+                    'which was unexpected.'
+            end if
+        else
+            error stop procedure // ' set11 had none bits set ' // &
+                'which was unexpected'
+        end if
+
+        call set10 % not()
+        do i=0, set10 % bits() - 1
+            if ( set10 % test(i) ) go to 500
+        end do
+
+        write(*,*) 'As expected set10 had no bits set.'
+
+        go to 510
+
+500     error stop procedure // ' against expectations set10 has ' // &
+            'at least 1 bit set.'
+
+510     continue
+
+        do i=0, set11 % bits() - 1
+            if ( .not. set11 % test(i) ) go to 600
+        end do
+
+        write(*,*) 'As expected set11 had all bits set.'
+
+        go to 610
+
+600     error stop procedure // ' against expectations set11 has ' // &
+            'at least 1 bit unset.'
+
+610     continue
+
+        do i=0, set10 % bits() - 1
+            if ( set10 % value(i) /= 0 ) go to 700
+        end do
+
+        write(*,*) 'As expected set10 had no bits set.'
+
+        go to 710
+
+700     error stop procedure // ' against expectations set10 has ' // &
+                    'at least 1 bit set.'
+
+710     continue
```

Comment:
These gotos and the following ones could be removed too, right?

Reply:
Done. Sorry I missed them.